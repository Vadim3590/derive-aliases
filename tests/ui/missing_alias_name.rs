pub mod derive_alias {
    derive_aliases::define! {
        = ::core::cmp::Eq, ::core::cmp::PartialEq
    // ^ missing name
        AndMore = ::std::hash::Hash, ..Ord, ::core::clone::Clone, ::core::marker::Copy, ::core::default::Default;
    }
}

fn main() {}
