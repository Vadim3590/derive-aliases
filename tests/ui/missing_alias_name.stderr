error: expected alias name (identifier)
 --> tests/ui/missing_alias_name.rs:3:9
  |
3 |         = ::core::cmp::Eq, ::core::cmp::PartialEq
  |         ^
